src/components/vertical-divider/vertical-divider.pw.tsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import VerticalDividerComponent from "./components.test-pw";
+import { verticalDividerComponent } from "../../../playwright/components/vertical-divider/index";
+import { VerticalDividerProps } from "./vertical-divider.component";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+const height = ["h", "height"];
+const pixels = [30, 50];
+const tintData: [VerticalDividerProps["tint"], string][] = [
+  [3, "rgb(8, 57, 78)"],
+  [57, "rgb(145, 167, 177)"],
+  [71, "rgb(181, 196, 202)"],
+  [98, "rgb(250, 251, 251)"],
+];
+const displayInlineVal = [true, false];
+
+test.describe("should check VerticalDivider component properties", () => {
```

Comment:
comment: I think the test descriptions could do with a tidy up here, the output in the console is a little confusing/ verbose